docs/topics/mta-rn-new-features-6-2-0.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-6-2-0_{context}"]
+= New features
+
+
+This section describes the new features of the Migration Toolkit for Applications (MTA) 6.2.0.
+
+.Integration with JIRA
+
+The integration of {ProductName} with Jira allows you to track and manage the whole migration process, and delegate implementation to external issue managers or task managers.
+
+To introduce changes to the applications in the portfolio, you can create issues in Jira and assign them to developers.
```

Comment:
@anarnold97 @dymurray I am not sure about this line: "and delegate implementation to external issue managers or task managers."
It is not clear what "delegate implementation" means.

Other than that, LGTM

Reply:
I would suggest to remove that line. The next one says that "you can create issues in Jira and assign them to developers."

Reply:
maybe `delegate management`? Otherwise yeah I think you can remove it.

Reply:
@anarnold97 Please remove only this part: `, and delegate implementation to external issue managers or task managers`
Thanks!